src/components/SelectBox.js

```diff
@@ -64,12 +65,14 @@ class SelectBox extends React.Component {
     return (
       <div style={{ width: '100%' }} id={id}>
         <div className='dropdown'>
-          <button className={`btn btn-default dropdown-toggle ${style['dropdown-button']} ${validationClass}`} type='button' data-toggle='dropdown' id={`${id}-button-toggle`}>
-            <span className={style['dropdown-button-text']} id={`${id}-button-text`} title={selectedItem && selectedItem.value}>
-              {selectedItem ? selectedItem.value : NOBREAK_SPACE}
-            </span>
-            <span className='caret' id={`${id}-button-caret`} />
-          </button>
+          <OverlayTooltip id={`${id}-selectbox-tooltip`} placement={'bottom'} tooltip={selectedItem ? selectedItem.value : ''}>
```

Comment:
I just really really want to drop this SelectBox and replace it with a PF-react version.

Reply:
Let's create separate issue for that,
would you like to open an issue for that and attach link to the PF component that you wants to replace this component?

Reply:
It would need to be a separate issue for sure.  I was just complaining.